api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -450,14 +450,15 @@ function dbRewardsReducer(validatorsDictionary, dbRewards) {
 function rewardReducer(network, validators, reward, reducers) {
   let parsedRewards = []
   Object.entries(reward.validators).forEach((validatorReward) => {
-    if (!validators[validatorReward[0]]) return
+    const validator = validators.find(({ operatorAddress }) => operatorAddress === validatorReward[0])
```

Comment:
how about transforming the validators to a dictionary first. should be a lot faster than doing a lookup for every validator